tests/RecoveryData.test.ts

```diff
@@ -12,1561 +12,1594 @@
-                        "address": "GDNODE4KTE7VQUHVBLXIGD7VEFY57X4XV547P72D37SDG7UEO7MWOSNY"
-                    },
-                    {
-                        "value": "20000000000000",
-                        "address": "GDNODE5T7TWJ2S4UQSTM7KDHU2HQHCJUXFYLPZDDYGXIBUAH3U3PJQC2"
-                    },
-                    {
-                        "value": "20000000000000",
-                        "address": "GDNODE6ZXW2NNOOQIGN24MBEZRO5226LSMHGQA3MUAMYQSTJVR7XT6GH"
-                    },
-                    {
-                        "value": "20000000000000",
-                        "address": "GDNODE7J5EUK7T6HLEO2FDUBWZEXVXHJO7C4AF5VZAKZENGQ4WR3IX2U"
-                    }
-                ],
-                "payload": ""
-            },
-            {
-                "type": 0,
-                "inputs": [],
-                "outputs": [
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    },
-                    {
-                        "value": "610000000000000",
-                        "address": "GCOQEOHAUFYUAC6G22FJ3GZRNLGVCCLESEJ2AXBIJ5BJNUVTAERPLRIJ"
-                    }
-                ],
-                "payload": ""
-            }
-        ],
-        "merkle_tree": [
-            "0x6314ce9bc41a7f5b98309c3a3d824647d7613b714c4e3ddbc1c5e9ae46db29715c83127ce259a3851363bff36af2e1e9a51dfa15c36a77c9f8eba6826ff975bc",
-            "0x7a5bfeb96f9caefa377cb9a7ffe3ea3dd59ea84d4a1c66304ab8c307a4f47706fe0aec2a73ce2b186a9f45641620995f8c7e4c157cee7940872d96d9b2f0f95c",
-            "0x788c159d62b565655d9f725786c38e6802038ee73d7a9d187b3be1c7de95aa0ba856bf81bb556d7448488e71f4b89ce6eba319d0536798308112416413289254"
+    [
```

Comment:
Please print out JSON Pretty and make it easier to compare with previous data.
https://jsonformatter.org/json-pretty-print